tests/wicked/basic/sut/t01_basic.pm

```diff
@@ -56,14 +55,14 @@ sub run {
         die "Wrong list of interfaces from wicked";
     }
     record_info('Test 6', 'Bring an interface down with wicked');
-    assert_script_run("ifdown $iface");
-    assert_script_run("ping -q -c1 -W1 -I $iface 10.0.2.2 2>&1 | grep -q ' Network is unreachable'");
+    $self->wicked_command('ifdown', $iface);
+    die('IP should not be reachable') if ($self->ping_with_timeout(ip => '10.0.2.2', timeout => 2));
     assert_script_run("! \$(ip address show dev $iface | grep -q 'inet')");
```

Comment:
Maybe we  can use `die if ($self->get_current_ip($iface));`.

Reply:
done